soccer/src/soccer/strategy/agent/position/runner.hpp

```diff
@@ -0,0 +1,55 @@
+#pragma once
+
+#include <cmath>
+
+#include <spdlog/spdlog.h>
+
+#include <rj_common/time.hpp>
+#include <rj_geometry/geometry_conversions.hpp>
+#include <rj_geometry/point.hpp>
+#include <optional>
+
+#include "position.hpp"
+#include "rclcpp/rclcpp.hpp"
+#include "rclcpp_action/rclcpp_action.hpp"
+#include "rj_msgs/action/robot_move.hpp"
+
+
+#include "planning/instant.hpp"
+#include "rj_common/field_dimensions.hpp"
+#include "rj_common/time.hpp"
+#include "rj_constants/constants.hpp"
+#include "rj_geometry/geometry_conversions.hpp"
+#include "rj_geometry/point.hpp"
+#include "waller.hpp"
```

Comment:
it is a good habit to go remove unnecessary includes once you're done writing a class. usually, extra includes are optimized away by the compiler, but why take the chance